src/basic/handlers/basic.ts

```diff
@@ -241,12 +291,14 @@ class BasicStepHandler implements FRUIStepHandler {
       this.resolve();
     } else if (!this.submit && isValid) {
       this.resolve();
+    } else if (rendererClassname === 'SelectIdPCallbackRenderer') {
+      this.resolve();
```

Comment:
If the renderer that triggered this change was SelectIdPCallbackRenderer, then just immediately resolve and call next. We don't have to worry about other callbacks and if they are valid.

Reply:
Yep, make sense.